spec/classes/init_spec.rb

```diff
@@ -81,10 +81,14 @@
               it { is_expected.to contain_yumrepo('cvmfs').with_baseurl('https://cern.ch/cvmrepo/yum/cvmfs/EL/7/x86_64') }
               it { is_expected.to contain_yumrepo('cvmfs-testing').with_baseurl('https://cern.ch/cvmrepo/yum/cvmfs-testing/EL/7/x86_64') }
               it { is_expected.to contain_yumrepo('cvmfs-config').with_baseurl('https://cern.ch/cvmrepo/yum/cvmfs-config/EL/7/x86_64') }
-            else
+            when '8'
               it { is_expected.to contain_yumrepo('cvmfs').with_baseurl('https://cern.ch/cvmrepo/yum/cvmfs/EL/8/x86_64') }
               it { is_expected.to contain_yumrepo('cvmfs-testing').with_baseurl('https://cern.ch/cvmrepo/yum/cvmfs-testing/EL/8/x86_64') }
               it { is_expected.to contain_yumrepo('cvmfs-config').with_baseurl('https://cern.ch/cvmrepo/yum/cvmfs-config/EL/8/x86_64') }
+            else
+              it { is_expected.to contain_yumrepo('cvmfs').with_baseurl('https://cern.ch/cvmrepo/yum/cvmfs/EL/9/x86_64') }
+              it { is_expected.to contain_yumrepo('cvmfs-testing').with_baseurl('https://cern.ch/cvmrepo/yum/cvmfs-testing/EL/9/x86_64') }
+              it { is_expected.to contain_yumrepo('cvmfs-config').with_baseurl('https://cern.ch/cvmrepo/yum/cvmfs-config/EL/9/x86_64') }
```

Comment:
I guess it would be easier to use `.with_baseurl("https://…/EL/#{facts[:os]['release']['major']}/x86_64")` instead of the when/else construct here, but wouldn't block on it, given it's a pattern that has been here before this PR

Reply:
I prefer the pain of the static string, feels like you are cheating the test if you use a variable.